plugins/net_plugin/net_plugin.cpp

```diff
@@ -1493,7 +1493,7 @@ namespace eosio {
           try {
             b = cc.fetch_block_by_id(blkid);
           } catch (const assert_exception &ex) {
-            elog( "caught assert on fetch_block_by_id, ${ex}",("ex",ex.what()));
+            elog( "caught assert on fetch_block_by_id, ${ex}",("ex",ex.to_string()));
```

Comment:
I think you would be better served using FC_CAPTURE_AND_LOG macro for here and all these catch blocks.

Reply:
right. and after I am done testing I am happy to make this change.